src/utils/sigma.utils.js

```diff
@@ -229,12 +229,16 @@
    * @param  {number} y1  The Y coordinate of the start point.
    * @param  {number} x2  The X coordinate of the end point.
    * @param  {number} y2  The Y coordinate of the end point.
+   * @param  {number} cc  The curvature coefficients.
    * @return {x,y}        The control point coordinates.
    */
-  sigma.utils.getQuadraticControlPoint = function(x1, y1, x2, y2) {
+  sigma.utils.getQuadraticControlPoint = function(x1, y1, x2, y2, cc) {
+	cc = $.extend({ x: 2, y: 4 }, cc);
```

Comment:
`$` is not available in sigma. Please replace `$.extend` by `sigma.utils.extend`.